### PR TITLE
Feature/improve tales header

### PR DESCRIPTION
**Pull request by daria31v**
add a property 'image' to - stories.ts and index.ts; 
change the default image to the current image of the story - [story].tsx

### CODE REVIEW

**Comment by met** [comment]
Good. It works. 👍

**Result can be check** on [Facebook sharing debugger](https://developers.facebook.com/tools/debug/)

E.g. https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fmovapp-d552dcmzc-ceskodigital.vercel.app%2Fuk%2Fkids%2Fstories%2Fpernikova-chaloupka

Just ignore warning _Missing Properties The following required properties are missing: og:url, og:type, fb:app_id_, that is not important to us now.

**Concernig the image size**, that is discussed on [Images in Link Shares](https://developers.facebook.com/docs/sharing/webmasters/images/) Facebook documentation.

-  Minimum dimensions are 600 x 315
-  Maximum file size 8 MB.

 So it looks we are inside limits with the size.
 
 **Concerning the code:** I would prefere not to add image field into Story interface. That is not necessary, image URL could be generated during use of SEO components, just easy by concatenate 'https://www.movapp.cz/kids/' + slug + '.jpg'

**Comment by jejdacz** [comment]
Lesson learned. Good job.

**Review by jejdacz — COMMENTED** [review]
Seems facebook doesn't like your idea. But I think It could be ok in case you can't use filenames and paths that way.

File: pages/kids/stories/[story].tsx [hunk 1]
```diff
@@ -36,13 +36,14 @@ const StoriesContainer = ({ story, phrases }: StoriesProps): ReactNode => {
...
+  const image_current_story = story.slug;
 
   return (
     <div className="bg-gradient-to-r from-[#fdf6d2] to-[#99bde4] -mb-8 -mt-2 md:p-12 w-full">
       <SEO
-        title={t(`seo.kids_page_storiesTitle.${getCountryVariant()}`)}
+        title={`${title_current + ' - '}` + t(`seo.kids_page_storiesTitle.${getCountryVariant()}`)}
         description={t(`seo.kids_page_storiesDescription.${getCountryVariant()}`)}
-        image="https://www.movapp.cz/kids/hrad.png"
+        image={`https://www.movapp.cz/kids/' + ${image_current_story} + '.jpg'`}
```
Comment (met) [inline pages/kids/stories/[story].tsx:46]: I see one apostrophe is missing here, please correct that @daria31v @jejdacz 

image

**Pull request merged**